src/components/cart/CartItem/CartItem.tsx

```diff
@@ -21,19 +23,29 @@ interface Props {
 const CartItem = ({ cartItem }: Props) => {
   const [startAnimation, setStartAnimation] = useState<boolean>(false);
   const [shrinkAnimation, setShrinkAnimation] = useState<boolean>(false);
+
   const { data: videoItem } = useFindOrGetVideo(cartItem.videoId);
 
   const { mutate: mutateDeleteFromCart, error } = useCartMutation();
 
+  const videoRemovedHotjarEvent = () => {
+    HotjarFactory.hotjar().event(HotjarEvents.VideoRemovedFromCart);
+  };
+
   const cartItemAnimate = () => {
     setStartAnimation(true);
+
     setTimeout(() => {
       setShrinkAnimation(true);
     }, 200);
 
     setTimeout(() => {
       mutateDeleteFromCart(cartItem.id);
     }, 600);
+
+    setTimeout(() => {
```

Comment:
Could we add this to the onSuccess of the cartMutation? Or pass it into the mutation. Not sure why the `setTimeout` is needed here